hw/arm/wr-arm.c

```diff
@@ -38,6 +38,34 @@ static void *wr_arm_get_dtb(const struct arm_boot_info *binfo)
     return ms->fdt;
 }
 
+static void wr_arm_create_virtio_transports(MachineState *machine)
+{
+    WrArmMachineState *s = WR_ARM_MACHINE(machine);
+    const char compat[] = "virtio,mmio";
+    int num_transports = wr_memmap[WR_VIRTIO].size / WR_VIRTIO_SZ;
```

Comment:
= 4. Seems like a sufficient number.

Reply:
I originally had number of transports specified as a #define but was worried about memory map conflicts if a big number was given. So i made it instead determine the number of virtio transports it can fit into the region size specified in the memory map for WR_VIRTIO.